concepticondata/conceptlists/Girard-1971-546.tsv

```diff
@@ -0,0 +1,547 @@
+ID	NUMBER	SPANISH	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	PROTO_ID	PROTO_CONCEPT
```

Comment:
We do not need "Proto-Concept" here, because in this list we only include proto-concepts anyway

Reply:
Then, neither Proto-ID, right?

Reply:
We keep it, but rename to "SOURCE_ID".